bluebottle/projects/models.py

```diff
@@ -705,6 +708,14 @@ class Meta(BaseProject.Meta):
             ('api_add_project', 'Can add projects through the API'),
             ('api_change_project', 'Can change projects through the API'),
             ('api_delete_project', 'Can delete projects through the API'),
+            ('api_read_projectdocument', 'Can view project documents through the API'),
+            ('api_add_projectdocument', 'Can add project documents through the API'),
```

Comment:
@gannetson not so sure about this but the project document ownership is based on the project ownership so the permissions needed to be here. It's a WIP